### PR TITLE
feat(InteractionResponses): add message parameter (v13)

### PR DESCRIPTION
**Pull request by jaw0r3k**
**Please describe the changes this PR makes and why it should be merged:**

**Status and versioning classification:**
Backboards #8773

### CODE REVIEW

File: src/structures/interfaces/InteractionResponses.js [hunk 1]
```diff
@@ -140,24 +147,25 @@ class InteractionResponses {
...
   }
 
   /**
-   * Deletes the initial reply to this interaction.
+   * Deletes a reply to this interaction.
    * @see Webhook#deleteMessage
+   * @param {MessageResolvable|'@original'} [message='@original'] The response to delete
    * @returns {Promise<void>}
    * @example
    * // Delete the reply to this interaction
```
Comment (MrMythicalYT) [inline src/structures/interfaces/InteractionResponses.js:161]:
```suggestion
   * // Delete the initial reply to this interaction
```

File: src/structures/interfaces/InteractionResponses.js [hunk 2]
```diff
@@ -114,23 +114,30 @@ class InteractionResponses {
   }
 
   /**
-   * Fetches the initial reply to this interaction.
+   * Fetches a reply to this interaction.
    * @see Webhook#fetchMessage
+   * @param {MessageResolvable|'@original'} [message='@original'] The response to fetch
    * @returns {Promise<Message|APIMessage>}
    * @example
    * // Fetch the reply to this interaction
```
[the remark this answers is not in the document]
Comment (jaw0r3k) [reply src/structures/interfaces/InteractionResponses.js:122]: isnt purpose of pr to fetch any reply?

File: src/structures/interfaces/InteractionResponses.js [hunk 3]
```diff
@@ -114,23 +114,30 @@ class InteractionResponses {
...
    *   .catch(console.error);
    */
-  fetchReply() {
-    return this.webhook.fetchMessage('@original');
+  fetchReply(message = '@original') {
+    return this.webhook.fetchMessage(message);
   }
 
   /**
    * Edits the initial reply to this interaction.
```
[the remark this answers is not in the document]
Comment (jaw0r3k) [reply src/structures/interfaces/InteractionResponses.js:132]: why?

[the remark this answers is not in the document]
Comment (MrMythicalYT) [reply src/structures/interfaces/InteractionResponses.js:132]: The "Edits the initial reply to this interaction.". Not sure why it's there

File: src/structures/interfaces/InteractionResponses.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (jaw0r3k) [reply src/structures/interfaces/InteractionResponses.js:122]: So if I add some id, it will be good?

[the remark this answers is not in the document]
Comment (MrMythicalYT) [reply src/structures/interfaces/InteractionResponses.js:122]: The example should be quick and easy, adding the ID isn't needed. You should just show that this fetches the initial reply

File: src/structures/interfaces/InteractionResponses.js (hunk 3 above)
Comment (jaw0r3k) [inline src/structures/interfaces/InteractionResponses.js:132]:
```suggestion
   * Edits the reply to this interaction.
```

File: src/structures/interfaces/InteractionResponses.js [hunk 4]
```diff
@@ -114,50 +114,58 @@ class InteractionResponses {
...
    */
-  fetchReply() {
-    return this.webhook.fetchMessage('@original');
+  fetchReply(message = '@original') {
+    return this.webhook.fetchMessage(message);
   }
 
   /**
-   * Edits the initial reply to this interaction.
+   * Options that can be passed into editReply.
```
Comment (kyranet) [inline src/structures/interfaces/InteractionResponses.js:132]: Same as https://github.com/discordjs/discord.js/pull/8840#discussion_r1027134946

**Review by kyranet — CHANGES_REQUESTED** [review]

File: src/structures/interfaces/InteractionResponses.js (hunk 4 above)
Comment (jaw0r3k) [reply src/structures/interfaces/InteractionResponses.js:132]:
```suggestion
   * Options that can be passed into {@link InteractionResponses#editReply}.
```

**Review by kyranet — APPROVED** [review]

File: typings/index.d.ts [hunk 5]
```diff
@@ -6057,7 +6057,9 @@ export type WebhookEditMessageOptions = Pick<
   WebhookMessageOptions,
   'content' | 'embeds' | 'files' | 'allowedMentions' | 'components' | 'attachments' | 'threadId'
 >;
-
+export interface InteractionEditReplyOptions extends WebhookEditMessageOptions {
+  message?: MessageResolvable | '@original';
+}
 export interface WebhookFetchMessageOptions {
```
Comment (iCrawl) [inline typings/index.d.ts:6063]:
```suggestion
}

export interface WebhookFetchMessageOptions {
```

**Review by iCrawl — APPROVED** [review]

**Pull request merged**